### ISSUE TITLE
Understanding SHI7EN parameters

### ISSUE THREAD

**Issue by pvangay** [knights-lab/shi7#29]
1. - What are we supposed to put for --r1 and --r2? Are these the expected identifiers for the two? So if my file is named "CS-137_S84_L001_R1_001.fastq", should I put in something like --r1 R1_001?

2. - Is --filter_length equivalent to create_trimmer_general_L150.sh? So if I used this script before, should I be changing this parameter to 150?

3. - Was 20 always the default for the trim quality? I thought it was 25 - but unfortunately I don't recall specifying this anywhere when I ran SHI7EN when it didn't have a wrapper.

**Comment by pvangay** [comment]
From Ben:

What are we supposed to put for --r1 and --r2? Are these the expected identifiers for the two? So if my file is named "CS-137_S84_L001_R1_001.fastq", should I put in something like --r1 R1_001? 
Specifying the R1/R2 are for the gottas split command, equivalent to split libraries from QIIME, and that documentation is currently not present. The default mode is paired_end, and will ignore the R1/R2. Gabe can comment more on how to use this.

Is --filter_length equivalent to create_trimmer_general_L150.sh? So if I used this script before, should I be changing this parameter to 150?
That is my intuition. I didn't use the old trimmer. Gabe can you answer this?

Was 20 always the default for the trim quality? I thought it was 25 - but unfortunately I don't recall specifying this anywhere when I ran SHI7EN when it didn't have a wrapper.
Again, Gabe can you answer this?

**Comment by kaiweiang** [comment]
FYI, the implementation of the gotta split in this wrapper has not been tested yet.

**Comment by pvangay** [comment]
Gabe's answers:
What are we supposed to put for --r1 and --r2? Are these the expected identifiers for the two? So if my file is named "CS-137_S84_L001_R1_001.fastq", should I put in something like --r1 R1_001?
-- no, R1 and R2 files are determined automatically. These are for split_libraries related work. Not necessary if working with files from UMGC.

Is --filter_length equivalent to create_trimmer_general_L150.sh? So if I used this script before, should I be changing this parameter to 150?
Yes

Was 20 always the default for the trim quality? I thought it was 25 - but unfortunately I don't recall specifying this anywhere when I ran SHI7EN when it didn't have a wrapper.
No, this 20 default is for a different parameter so leave it as is. Internally, the overall trimming quality is set to 30 (not settable from user params)
